app/views/guests/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1> Guest Page </h1>
+<div id="Facts">
+<h2>Name: <%= @guest.name %></h2>
+</div>
+
+<div id="Rooms">
+<h3> Rooms </h3>
+<% @rooms.each do |room| %>
+<div id="<%= room.suite %>">
+  <p>Suite: <%= room.suite %></p>
+  <p>Rate: <%= room.rate %></p>
+  <p>Hotel: <%= room.hotel.name %></p>
+  <p>------------------------------</p>
+</div>
+<% end %>
+</div>
+
+<div id="Add_room">
+<%= form_with url: "/guests/#{@guest.id}/rooms", method: :post do |form| %>
+    <p> <%= form.label :room_id, "Please enter a valid room ID:" %> </p>
+    <p> <%= form.text_area :room_id %> </p>
+    <p> <%= form.submit "Submit" %> </p>
+<% end %>
+</div>
```

Comment:
This all looks good! One thing we could do to make your route on line 19 more RESTful is to use the route 'POST /guest_rooms'. However, that would require that you use a hidden field in your form to pass through the guest id within the body of the request (along with the room_id).